include/config.h

```diff
@@ -118,6 +123,14 @@
 // -- SPECIFIC OBJECT SETTINGS --
 // Allow for retries on collecting the remaining blue coins from a blue coin switch
 #define BLUE_COIN_SWITCH_RETRY
+// The number of chain balls the Chain Chomp has.  Vanilla is 5.
+#define CHAIN_CHOMP_NUM_SEGMENTS 5
+
+// The number of parts Pokey has, including the head. Vanilla is 5, max is 30.
+#define POKEY_NUM_SEGMENTS       5
+
+// The number of segments Wiggler has, not including the head. Vanilla is 4.
+#define WIGGLER_NUM_SEGMENTS     4
```

Comment:
these seem like "top of the file" defines defines for a separate file. I know we _are_ going to split the config soon. so maybe not a huge deal now